packages/material-ui-lab/src/ToggleButton/ToggleButton.d.ts

```diff
@@ -1,17 +1,29 @@
-import { ButtonBaseClassKey, ExtendButtonBase } from '@material-ui/core/ButtonBase';
-import { SimplifiedPropsOf } from '@material-ui/core/OverridableComponent';
+import {
+  ButtonBaseClassKey,
+  ExtendButtonBase,
+  ExtendButtonBaseTypeMap,
+} from '@material-ui/core/ButtonBase';
+import { OverrideProps } from '@material-ui/core/OverridableComponent';
 
-declare const ToggleButton: ExtendButtonBase<{
-  props: {
+export type ToggleButtonTypeMap<
+  P = {},
```

Comment:
Any reason we use default types here? [`Button.d.ts`](https://github.com/mui-org/material-ui/blob/d8dfd09320b04d39ebedad7f405ac4ef107b430b/packages/material-ui/src/Button/Button.d.ts#L5) does not have default types.

Reply:
Gives the `defaultComponent` more meaning. Let's leave it just in case we get partial type application.